README.adoc

```diff
@@ -49,13 +50,18 @@ exist in the Kubernetes namespace:
 == Support
 
 Please only report bugs via
-https://github.com/trento-project/trento/issues[GitHub issues]; for any
+link:https://github.com/trento-project/trento/issues[GitHub issues]; for any
 other inquiry or topic use
-https://github.com/trento-project/trento/discussions[GitHub discussion].
+link:https://github.com/trento-project/trento/discussions[GitHub discussion].
 
 == Contributing
 
-See xref:CONTRIBUTING.adoc[how to Contribute]
+ifdef::site-gen-antora[]
```

Comment:
This is needed in order to make GitHub Preview and Antora Links both working. 

Otherwise there is a resoloution conflict with [:relfileprefix: docs/]